ui/src/api/index.ts

```diff
@@ -53,15 +53,20 @@ export class Hub implements Api {
 
   async resources() {
     try {
-      return axios.get(`${API_URL}/resources`).then((response) => response.data);
+      return await axios
```

Comment:
is this because of `npm run lint` or because of your editor settings

Reply:
no, this is for resolving the promise